lib/cretonne/meta/isa/intel/registers.py

```diff
@@ -46,10 +46,13 @@
         names=['eflags'])
 
 GPR = RegClass(IntRegs)
+# Certain types of deref encodings cannot be used with all registers.
+#   R13/RBP cannot be used with zero-offset load or store instructions.
+#   R12 cannot be used with a non-SIB-byte encoding of all derefs.
+GPR_DEREF_SAFE = GPR.without(GPR.r12)
```

Comment:
Should this exclude `GPR.rsp` too?

Reply:
Yes indeed. I misunderstood the Intel manual's explanation of this. Just pushed a new commit with that fix.

However, I see this is already merged. I'm not sure how to reopen a PR from the same branch, if that's even possible.

Reply:
Not sure. I'll just add it. Sorry for jumping the gun on the merge.